test/unit/validation/test_validation_detect_ongoing_zero_flatliner.py

```diff
@@ -14,6 +14,10 @@
     start=now - timedelta(minutes=180), end=now, freq="0.25H"
 )
 
+four_hour_range_predict_setting = pd.date_range(
+    start=now - timedelta(minutes=180), end=now + timedelta(minutes=60), freq="0.25H"
+)
+
```

Comment:
It's better to move this code into the test itself. If for some reason there is big delay between initializing this module and running the test, the datetime will not have the correct value anymore.